src/powershell/install.sh

```diff
@@ -221,25 +221,31 @@ install_using_github() {
         echo "${powershell_archive_sha256} *${powershell_filename}" | sha256sum -c -
     fi
     tar xf "${powershell_filename}" -C "${powershell_target_path}"
-    ln -s "${powershell_target_path}/pwsh" /usr/local/bin/pwsh
+    chmod 755 "${powershell_target_path}/pwsh"
+    ln -sf "${powershell_target_path}/pwsh" /usr/bin/pwsh
```

Comment:
We have intentionally attempted to install binaries in `/usr/local/bin` due to following differences 👇 

/usr/local/bin:

- Typically used for software installed manually by the system administrator.
- Preferred for custom or third-party software installations.
- Keeps the system's default binaries (/usr/bin) separate from manually installed binaries.
- Higher precedence in the PATH variable on many systems, meaning binaries here are found first.

/usr/bin:

- Used for binaries provided by the system's package manager.
- Standard location for system-wide binaries.
- Modifying this directory can potentially interfere with system updates or package management.

Can you help me understand your thoughts for this change? Thanks!

Reply:
The intention is to mimic the exact same setup from Debian repository packages.
I am familiar with the differences between `/usr/local/bin` and `/usr/bin`. In my opinion, we are not a typical user installing something manually, we are acting similar to package maintainers for what `/usr/bin` is exactly for.

On top of that, the GitHub installation is more an automatic fallback where people might still expect the very same outcome as if they had been lucky to have a packaged version available.